Cart/Cart.css

```diff
@@ -0,0 +1,72 @@
+.cart-component {
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  margin: 1rem;
+  padding: 1rem;
+  flex-wrap: wrap;
+}
+
+.product-detail {
+  margin: auto;
+}
+
+.product-name {
+  font-weight: bold;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  margin-top: 0rem !important;
+}
+
+.qty-btn {
```

Comment:
could a 
display flex 
align-items : center;

help take the minus sign to the center of the button element ? hope it helps, everything else looks amazing !